validation-test/stdlib/ArrayNew.swift.gyb

```diff
@@ -1110,6 +1110,15 @@ ArrayTestSuite.test("BridgedToObjC.Nonverbatim.RoundtripThroughSwiftArray") {
   }
 }
 
+ArrayTestSuite.test("append(contentsOf: NSArray)") {
+  // A stray runtime `is` test caused this particular operation to fail in 5.3.
+  // rdar://70448247
```

Comment:
Note: I did not add an availability test here on the theory that we aren't running these tests with stdlib binaries built from the 5.3 release. If this proves wrong, we may need to add one later.